src/main/java/io/temporal/samples/hello/HelloActivity.java

```diff
@@ -74,39 +104,80 @@ public String getGreeting(String name) {
     }
   }
 
+  /**
+   * Implementation of our workflow activity interface. It overwrites our defined composeGreeting
+   * activity method.
+   */
   static class GreetingActivitiesImpl implements GreetingActivities {
     @Override
     public String composeGreeting(String greeting, String name) {
       return greeting + " " + name + "!";
     }
   }
 
+  /**
+   * With our Workflow and Activities defined, we can now start execution. The main method is our
+   * workflow starter.
+   */
   public static void main(String[] args) {
-    // gRPC stubs wrapper that talks to the local docker instance of temporal service.
+
+    /*
+     * Define the workflow service. It is a gRPC stubs wrapper which talks to the docker instance of
```

Comment:
I would remove word docker from here, we should be able to run samples against our cloud service in the future.